packages/uix-core/src/tunnel/tunnel.test.ts

```diff
@@ -185,14 +203,18 @@ describe("static Tunnel.toIframe(iframe, options)", () => {
       });
       localTunnel.on("connected", connectMessageHandler);
       await wait(100);
-      fireEvent(
-        window,
-        new MessageEvent("message", {
-          data: messenger.makeOffered("iframe-test-1"),
-          origin: loadedFrame.src,
-          source: loadedFrame.contentWindow,
-        })
+      window.postMessage(
+        messenger.makeOffered("iframe-test-1"),
+        loadedFrame.src
       );
+      // fireEvent(
+      //   window,
+      //   new MessageEvent("message", {
+      //     data: messenger.makeOffered("iframe-test-1"),
+      //     origin: loadedFrame.src,
+      //     source: loadedFrame.contentWindow,
+      //   })
+      // );
```

Comment:
Do we need to keep the comment?

Reply:
It's in a test, so I'm saying yes for now :)